modules/tutorials/pages/java-tutorial/first-couchbase-application.adoc

```diff
@@ -0,0 +1,191 @@
+:page-pagination:
+
+= Writing your first Couchbase application
+
+include::partial$tutorial-globals.adoc[]
+
+:crud-footnote: footnote:[Create Read Update Delete]
+
+:description: pass:q[Following on from xref:install-couchbase-java-sdk.adoc[], this tutorial will show you how to connect to Couchbase and add records to the database using the SDK.]
+
+[abstract]
+{description}
+
+== Introduction
+Working with any database SDK usually involves the following operations:
+
+. Connect to the database
+. Carry out CRUD{crud-footnote} operations
```

Comment:
Footnote feels a bit heavy. Is there a way to use `<abbr>` tag?
Or just put it in parentheses?

Reply:
Could use the <abbrev> tag, but I'm not keen on putting HTML in the Asciidoc.  Might have to see if we can get this:

https://github.com/asciidoctor/asciidoctor/issues/252

moved on a bit. In the meantime, I'll leave it as a footnote.